compute/containers/quickstart.mdx

```diff
@@ -14,7 +14,7 @@ dates:
   posted: 2021-05-26
 ---
 
-A container is a package of software that includes all dependencies: code, runtime, configuration, and system libraries so that the software can run on any host system. The Scaleway Elements Containers platform provides you with custom Docker images that are entirely handled for you in the cloud. You can use your favorite technologies such as Django or Ruby On Rails for your containers. We show you how to create a Containers namespace, deploy a container, and delete it when you are done.
+A container is a package of software that includes all dependencies: code, runtime, configuration, and system libraries so that the software can run on any host system.  The Scaleway Elements Containers platform provides you with containerized stateless applications deployed in the cloud. You can use your favorite technologies such as Django or Ruby On Rails for your containers. We show you how to create a Containers namespace, deploy a container, and delete it when you are done.
```

Comment:
```suggestion
A container is a package of software that includes all dependencies: code, runtime, configuration, and system libraries so that the software can run on any host system. The Scaleway Elements Containers platform provides you with containerized stateless applications deployed in the cloud. You can use your favorite technologies such as Django or Ruby On Rails for your containers. We show you how to create a Containers namespace, deploy a container, and delete it when you are done.
```